docs/src/routes/docs/_layout.svelte

```diff
@@ -1,11 +1,29 @@
 <script>
+  import { onMount } from 'svelte';
+  import { Button } from 'attractions';
+  import { HomeIcon, ArrowUpIcon } from 'svelte-feather-icons';
   import Header from 'src/containers/docs/header.svelte';
   import MobileNavigation from 'src/containers/docs/mobile-navigation.svelte';
   import DesktopNavigation from 'src/containers/docs/desktop-navigation.svelte';
-  import { HomeIcon } from 'svelte-feather-icons';
 
   export let segment;
 
+  function scrollToTop() {
+    window.scrollTo(0, 0);
+    // TODO: manage focus
+  }
+
+  let scrollbarVisible;
+
+  function updateScrollbarVisibility() {
+    if (typeof window === 'undefined') {
+      return;
+    }
+    scrollbarVisible = window.innerHeight < document.body.scrollHeight;
+  }
+  onMount(updateScrollbarVisibility);
+  $: updateScrollbarVisibility(segment);
```

Comment:
That doesn't seem to be working when I navigate using the buttons instead of the address bar. If I go to [installation](https://illright.github.io/attractions/202/docs/installation) and then click on the home button in the breadcrumb, the button is still visible even though it is not needed